src/utils/calculateClicks.js

```diff
@@ -0,0 +1,17 @@
+import _ from "lodash";
+
+const calculateClicks = (campaignOrChannel, data) => {
+  const receivedData = data;
+  const clicks = _.filter(receivedData, element => {
+    if (
+      _.isEqual(element.campaign, campaignOrChannel) ||
+      _.isEqual(element.channel, campaignOrChannel)
+    ) {
+      return element;
+    }
+  });
+  const totalClicks = _.sumBy(clicks, "clicks");
+  return totalClicks;
```

Comment:
if u follow my request then you can avoid a `return` statement for this entire function

Reply:
0ad76fe2fd3d0221c48e0a55eda118f4fb6947a2